index.js

```diff
@@ -35,9 +35,11 @@ function organizeMoonsByPlanet(moons) {
 
   You should not need to make any other changes to the code.
 */
-function getClosestToPlanet(moons) {
-  let closest = moons[0];
-  const rest = moons.slice(1);
+//function getClosestToPlanet(moons) 
+const getClosestToPlanet = (moons) => {
+  //let closest = moons[0];
+  //const rest = moons.slice(1);
+  let[closest,...rest] = moons;
```

Comment:
Well done!

Might seem minor, but the spaces after commas (skipped in this line and the destructuring assignment in the previous function) are very important for readability. Without the comma, it looks like one thing. If code is written with standard whitespace usage, you can skim it, looking at the structure of the code, and know how many properties are being destructured, how many variables declared, how many parameters defined, etc.

Try to get yourself into professional habits over the next year. All the professional habits. Become _such a pro_.